src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/TicketSearchService.cs

```diff
@@ -43,7 +44,12 @@ public TicketSearchService(IBus bus, ITicketInvoiceSearchService invoiceSearchSe
             IEnumerable<Invoice>? response = await _invoiceSearchService.SearchAsync(ticketNumber, issuedTime, cancellationToken);
 
             // TCVP-2651 Filter results to only MVA/MVAR violation tickets
-            var invoices = response.Where(_ => _.Act == _mva || _.Act == _mvar).ToList();
+            var invoices = response.Where(_ => _.Act == _mva || _.Act == _mvr || _.Act == _mvar).ToList();
+            
+            var droppedInvoices = response.Where(_ => _.Act != _mva && _.Act != _mvr && _.Act != _mvar).ToList();
+            if (droppedInvoices.Count != 0) {
+                _logger.LogDebug("Dropped {Count} non MVA/MVAR violation ticket(s) from the RSI search results", droppedInvoices.Count);
```

Comment:
Would be useful to consider changing the logging to use the source generator one like in this file https://github.com/bcgov/jag-traffic-courts-online/blob/263d460622f7552708372dd6e934dd1ec64eea1d/src/backend/TrafficCourts/Workflow.Service/Consumers/DisputeApprovedConsumer.cs#L157-L160.  We want to add additional properties in a standard way. We are not logging the ticket number or time if we wanted to troubleshoot this.